job-utilities/job-router/README.md

```diff
@@ -0,0 +1,39 @@
+# job-router
+
+Python application to route scheduled jobs to ECS services on AWS infrastructure
+
+## Building
+
+A docker image for local use can be built using 'docker build --platform linux/amd64 -t image-name:image-tag .'
+Putting together a package for deployment to Lambda is done with the following steps:
+
+mkdir package
+pip install --target ./package -r requirements.txt
+cd package
+zip -r ../deployment_package.zip .
+cd ..
+zip deployment_package.zip lambda_function.py
+
+## Deploying
+
+A manual deployment of the zip package is easy. Simply go to the job-router Lambda and upload the zip package of code to update it.
+
+## Testing/Running locally
+
+The job-router Lambda can be tested locally by building the docker image and using the run-docker.sh script. Testing from local is fine,
+but there's not much reason to actually use this code as a regular part of local dev, since it just routes to endpoints that are directly
+available to local development.
```

Comment:
Can we put into dev-system when we start the CMR to be able to run the scheduler automatically if the developer wants it? Right now we can set modes in dev-system/dev/user.clj and then depending on the modes it can automatically start and calls bootstrap when the event fires.

Reply:
That's one of the things I realized I should look at adding in. I'll be working on that tomorrow

Reply:
Another ticket has been made for looking into this